drivers/flash/flash_simulator_stub/Kconfig

```diff
@@ -0,0 +1,25 @@
+# Flash simulator config
+
+# Copyright (c) 2018 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig FLASH_SIMULATOR_STUB
+	bool "Flash simulator stub"
```

Comment:
Is there a good reason why this symbol is having a prompt ?

Shouldn't it be prompt less and then internal Kconfig can select this symbol when mcuboot is enabled for thingy53.
(Other board may do similar when they start supporting dual image updates)

Reply:
It is no for mcuboot but for the DFU app. Mcuboot is using upstream flash_simulator

Reply:
> 
> It is no for mcuboot but for the DFU app. Mcuboot is using upstream flash_simulator

@nvlsianpu I assume that what mcuboot will be using will depend on what we choose to do in this commit (which is a fromlist):
https://github.com/nrfconnect/sdk-mcuboot/pull/175/commits/e67d3dc27be1bb8b9f36e607cb0cecf78e448d65

as well as the possibility in sdk-nrf to choose alternative Kconfig settings for mcuboot as a child image.

Reply:
yes, but the DFU is only enabled if mcuboot is enabled, afaik.

Meaning this symbol is not something a user / conf file should set, but should be automatically selected when mcuboot and DFU for dual image update is enabled.
And this is being enabled in the app.

Reply:
> 
> 
> yes, but the DFU is only enabled if mcuboot is enabled, afaik.
> 
> Meaning this symbol is not something a user / conf file should set, but should be automatically selected when mcuboot and DFU for dual image update is enabled.
> And this is being enabled in the app.

You mean that you want to hide the Kconfig options completely?